src/schema/nmdc.yaml

```diff
@@ -1495,9 +1495,22 @@ enums:
       Metagenome Bins:
         description: Metagenome bin contigs fasta
 
+      Metagenome Bins File: 
```

Comment:
I want to make sure we are differentiating between these two similarly named enumerations:

- Metagenome Bins:
- Metagenome Bins File:

Reply:
Yeah, I wouldn't mind moving Metagenome Bins File to Metagenome Bins Compression File or Metagenome Bins Archive -- something that captures a collection of many metagenome bins

Reply:
I'll update this in a new branch.